packages/firestore/src/local/indexeddb_index_manager.ts

```diff
@@ -62,10 +62,11 @@ export class IndexedDbIndexManager implements IndexManager {
         this.collectionParentsCache.add(collectionPath);
       });
 
-      return collectionParentsStore(transaction).put({
+      const collectionParent: DbCollectionParent = {
         collectionId,
         parent: encode(parentPath)
-      });
+      };
+      return collectionParentsStore(transaction).put(collectionParent);
```

Comment:
FWIW this seems like another case we could detect.  Since collectionParent gets passed to a function that accepts `DbCollectionParent`...

Reply:
That's definitely true. I haven't seen any other cases like this yet, but I can spend some time on this if there are more (or I run out of other quirks).